projects/documentation/content/guides/configuring-openwc.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
This content begs the question of whether we should contribute back to the generator and/or build/extend our own, but it's great for now, as that's more work than we want to pick up right now. However, the second question in [this issue](https://github.com/open-wc/create/issues/80) aligns really closely with goals or our project if we ever decide to go down that path.